### DISCUSSION TITLE
How to use adsense??

### DISCUSSION THREAD

**Discussion by devwontory** [CaiJimmy/hugo-theme-stack#716] [Q&A] [answered]
i put my adsense code in script.html and requested approvement.
i checked in developer mode and the code was working.
but google adsense rejected my request. the reason was "connecting site".

help me please

**Comment by rmdhnreza** [comment, marked as answer]
Screenshot would be helpful.

Make sure your site is **Ready** under **Sites** menu on **AdSense Dashboard.**
